app/controllers/decidim/friendly_signup/validator_controller.rb

```diff
@@ -7,6 +7,7 @@ class ValidatorController < ApplicationController
 
       def validate
         @form = form(Decidim::RegistrationForm).from_params(params)
+        @form.errors.add(:password, :password_blank) if params[:attribute] == "password" && @form.password.blank?
```

Comment:
Why checking here if you added the check in the UserAttributeValidator ?

Reply:
Actually I think I could remove it, just probably put this one before the other, I'll check if that works and i'll remove it if it's useless, There are multiple verifications because when we confirm the form, we also need this verification to be applied, which was not if I'd put only this one if validator_controller, but I'll check and come back to you

Reply:
You were right, it was actually too much and not useful at all ! Thanks